static/js/src/entry/user-portal/components/Icon.vue

```diff
@@ -1,6 +1,12 @@
 <template>
-  <span class="c-icon" :class="`t-size-${display.size}`">
-    <svg aria-hidden="true"><use :xlink:href="`#${name}`" /></svg>
+  <span
+    class="c-icon"
+    :class="`t-size-${mergedDisplay.size} has-text-${mergedDisplay.color}`"
```

Comment:
I always like the escape hatch of being able to sprinkle in those extra classes, but I'm all for keeping it simple until we need it.